geotrellis/geotrellis-server
# Add filter, se, sld, and wfs schemas

## Overview

This PR adds `sld` types to the `opengis` package. It's not important that it's merged as much as it's important to prove that it's possible for an upcoming pr in Franklin.

### Checklist


### Notes

Thanks @pomadchin for help solving the schema version challenge.

~Apparently `RasterSymbolizer` isn't included in the generated types for some reason :thinking:. I'm not sure where it went. It has to be in the xsd schemas somewhere though~ It's in the "Symbology Encoding" (`se`) spec

## Testing Instructions

- `++ 2.12.10; opengis/publishLocal` (or another scala version that you have a convenient ammonite repl for)
- start an ammonite repl with a Scala version matching the version you published locally
- import the ivy artifact you just published:
```
import $ivy.`com.azavea.geotrellis::geotrellis-server-opengis:3.4.0-10-b73e8fb7-SNAPSHOT`
```
- `import opengis.sld.<TAB>` and check out what's available, `import opengis.se.<TAB>` and check out what's available
